### DISCUSSION TITLE
Universal translation style guide

### DISCUSSION THREAD

**Discussion by tesseralis** [reactjs/reactjs.org-translation#230] [Team Posts]
Does anyone have any thoughts on rules that should be true for *all* translations? So far I'm thinking:

* leave code blocks untranslated
* links to references (MDN, wikipedia) should go to the translated language, not the English version.
* Some words, like "React", should not be translated.

Anyone have any other suggestions?

**Comment by carburo** [comment]
Do you think it's ok to translate comments in code blocks?

**Comment by tesseralis** [comment]
Yeah! That should be fine.

**Comment by GasimGasimzada** [comment]
Do you think props and state should be translated? We have translated them to our language as a test and read through some of the docs. These terms make things very confusing. So, we are thinking of keeping "props" as is but translate properties (not short version) as "props" is almost like a reserved word. What do you think?
